ha/resource/sspl

```diff
@@ -141,7 +141,7 @@ stateful_demote() {
 }
 
 stateful_promote() {
-    sleep 5
+    sleep 8
```

Comment:
Proper fix would be implemented through https://jts.seagate.com/browse/EOS-13834